build.gradle

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Gradle build script for the data portability project.
+ *
+ * Try on *Nix:
+ *
+ * ./gradlew clean jar
+ *
+ * Or on Windows:
+ *
+ *  gradlew clean jar
+ *
+ */
+buildscript {
+
+    repositories {
+        jcenter()
+        mavenLocal()
+    }
+}
+
+def sourceProjects() {
+    // define projects containing source code
+    return subprojects.findAll { new File(it.projectDir, 'src').exists() }
+}
+
+configure(sourceProjects()) {
+    repositories {
+        jcenter()
+        mavenLocal()
+    }
+
+    apply plugin: 'java'
+    apply plugin: 'maven-publish'
```

Comment:
Should we add a comment that we are currently publishing to maven until we transition to fully run on gradle?